### DISCUSSION TITLE
Does Inter v4 include spacing of Inter Tight for display sizes?

### DISCUSSION THREAD

**Discussion by ms22421** [rsms/inter#588] [General]
I am new to Inter as a font. I see that there is an "Inter Tight" font available on github, however it does not receive updates.

I also see that there was previously an Inter Display.

Now I see that Inter Variable v4 is intended to change the kerning/optical size at larger formats, similar to other variable fonts.

My question is will Inter Variable be interpolating to Inter Tight spacing at large formats, or Inter Display, or something else?

I like using Inter Tight for headings because of the saved space, but if I can use just Inter Variable then I would do that. Currently I'm not sure.

**Comment by kenmcd** [comment]
> My question is will Inter Variable be interpolating to Inter Tight spacing at large formats, or Inter Display, or something else?

Something else.
There is no direct comparison/equivalent to the older fonts.
Your best bet going forward is to use the new Inter 4 variable.

The Inter 4 optical size axis at the maximum of 28 is as "tight" as it gets now.
That is not quite as tight as Inter Tight (or the really old v3.012 Display).
But the newer v4 has many fixes, many new features, and will be maintained going forward.

Inter Tight was a special build for Google Workspace.
They now maintain it, and only they will make changes/improvements, if any.
It is available on GF: https://fonts.google.com/specimen/Inter+Tight
And it does have slightly different vertical metrics.

The Inter 4.0-beta9g includes some static fonts you can test.
The current static Display fonts in the extras folder are created at optical size 28.
So you can test those fonts to see if that works for you going forward.
